### DISCUSSION TITLE
Hide add-on configuration if no configuration for the add-on

### DISCUSSION THREAD

**Discussion by ludeeus** [home-assistant/frontend#8130] [Supervisor] [answered]
_Quote from: @frenck in <https://github.com/home-assistant/developers.home-assistant/pull/771>_

> For a future improvement, it would be nice if we could expose if the add-on has options (and thus a schema) and hide the config from the UI based on that.

**Comment by frenck** [comment]
Voting this one of course 😉

**Comment by ludeeus** [comment, marked as answer]
Proposal for implementation <https://github.com/home-assistant/frontend/pull/8131>

_Edit: The proposal was accepted and has been merged into the codebase._
